kong.conf.default

```diff
@@ -770,6 +770,12 @@
                                  # If set to 0 (default), such cached entities
                                  # or misses never expire.
 
+#db_cache_neg_ttl = 0            # Time-to-live (in seconds) of database misses
+                                 # where no entity is returned and nil is cached.
+                                 #
+                                 # If set to 0 (default), such nil entities
```

Comment:
Well done adding the docs! With the suggested changes below, this needs to be changed to mention that the default is to inherit the value of `db_cache_ttl`.